user_tools/src/spark_rapids_pytools/resources/templates/cluster_template/emr.ms

```diff
@@ -0,0 +1,30 @@
+{
+	"Cluster": {
+	  "Id": "j-123456789",
+	  "Name": "default-cluster-prop",
+	  "Status": {
+	    "State": "TERMINATED"
+	  },
+	  "Ec2InstanceAttributes": {
+	    "Ec2AvailabilityZone": "us-west-2a"
+	  },
+	  "InstanceGroups": [
+	    {
+	      "Id": "ig-123456789012e",
+	      "Name": "CORE",
+	      "Market": "ON_DEMAND",
+	      "InstanceGroupType": "CORE",
+	      "InstanceType": {{{ EXECUTOR_INSTANCE }}},
+	      "RequestedInstanceCount": {{ NUM_EXECUTOR_NODES }}
+	    },
+	    {
+	      "Id": "ig-123456789012d",
+	      "Name": "MASTER",
+	      "Market": "ON_DEMAND",
+	      "InstanceGroupType": "MASTER",
+	      "InstanceType": {{{ DRIVER_INSTANCE }}},
+	      "RequestedInstanceCount": 1
```

Comment:
Should this be `NUM_EXECUTOR_NODES` ?

Reply:
Currently we support only single driver node for all CSPs. We can extract number of driver nodes when we add support for extracting memory, cpu for driver node for OnPrem

Reply:
Oh I see. that's driver. not executors.
One thing could be better if the template does not have cardcoded value. then it is easier to update the code later with the correct value. but I am ok as it is for now.

Reply:
Sure. I will add this in the template